weblate/addons/base.py

```diff
@@ -119,10 +140,11 @@ def configure(self, configuration) -> None:
         self.post_configure()
 
     def post_configure(self, run: bool = True) -> None:
-        component = self.instance.component
+        # What target to have in site-wide add-ons?
```

Comment:
In `post_configure`

```
# Where to log for site-wide add-ons?
target = self.instance.component or self.instance.project
```

Reply:
I think it should use a global logger, something like:

```py
import logging

logging.getLogger("weblate.addons")
```

The annoying thing is that `LoggerMixin` (used in `Project` and `Component`) has a different API.